src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyTotal = 0;
+        int buyTotal = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                String operation = parts[0];
+                int amount = Integer.parseInt(parts[1]);
+
+                if (operation.equals("supply")) {
+                    supplyTotal += amount;
+                } else if (operation.equals("buy")) {
+                    buyTotal += amount;
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Exception", e);
```

Comment:
The exception message "Exception" is not very informative. Consider providing a more detailed message, such as "Can't read data from the file " + fromFileName.